src/platform/plugins/shared/discover/public/context_awareness/profile_providers/common/patterns/pattern_cell_renderer.tsx

```diff
@@ -20,77 +24,94 @@ interface Props {
 }
 
 export const PatternCellRenderer: FC<Props> = ({ pattern, isDetails, defaultRowHeight }) => {
-  const { euiTheme } = useEuiTheme();
-
-  const keywordStyle = useMemo(
-    () => ({
-      marginRight: euiTheme.size.xs,
-      marginBottom: `calc(${euiTheme.size.m} / 2)`,
-      display: 'inline-block',
-      padding: `${euiTheme.size.xxs} ${euiTheme.size.s}`,
-      backgroundColor: euiTheme.colors.lightestShade,
-      borderRadius: euiTheme.border.radius.small,
-      color: euiTheme.colors.textPrimary,
-      fontSize: euiTheme.size.m,
-    }),
-    [euiTheme]
-  );
+  const styles = useMemoCss(componentStyles(defaultRowHeight));
```

Comment:
Oops, I gave bad advice here because I overlooked that we're using dynamic styles based on `defaultRowHeight`. This returns a new object on each render so it will break memoization. I think the official recommendation is to inline dynamic styles and memoize static styles that use `euiTheme`, but what you had before was better than now.

Reply:
Changed in https://github.com/elastic/kibana/pull/222871/commits/e697375aa49b66e759ff321ee0817365bde11390